### PR TITLE
docs: update templates for starlark and go

### PR DESCRIPTION
**Pull request by JordanSussman**
supersede #185

### CODE REVIEW

File: content/templates/tutorials/go/vars_platform.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+## Overview
+
+Platform variables can be referenced with the following syntax:
+
+`{{ vela "<environment variable>" }}`
+
+## Examples
+
+- `{{ vela "VELA_FULL_REPO_NAME" }}` equates to the `VELA_REPO_NAME` environment variable
```
Comment (plyr4) [inline content/templates/tutorials/go/vars_platform.md]: should `VELA_REPO_NAME ` be `VELA_FULL_REPO_NAME`
```suggestion
- `{{ vela "VELA_FULL_REPO_NAME" }}` equates to the `VELA_FULL_REPO_NAME` environment variable
```

**Review by plyr4 — CHANGES_REQUESTED** [review]

File: content/templates/tutorials/go/vars_platform.md (hunk 1 above)
Comment (JordanSussman) [reply content/templates/tutorials/go/vars_platform.md]: You are correct.

**Review by kneal — APPROVED** [review]
LGTM 🐬

**Comment by kneal** [comment]
@JordanSussman Will close the draft

**Review by jbrockopp — APPROVED** [review]
LGTM

**Review by plyr4 — APPROVED** [review]

**Pull request merged**